src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: ['Under construction'],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
```

Comment:
- [ ] Good job creating categoriSlice, however, let's Include a reducer that `checks the status` as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer%20that%20checks%20the%20status%20and%20always%20returns%20%22Under%20construction%22%20(the%20initial%20state%20should%20check%20to%20that%20string)) 👍 . Let's take  an example form this:

```javascript
import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Keep doing good job 💪